Sources/DatadogObjc/DatadogConfiguration+objc.swift

```diff
@@ -379,6 +396,11 @@ public class DDConfigurationBuilder: NSObject {
         }
     }
 
+    @objc
+    public func set(mobileVitalsFrequency: DDVitalsFrequency) {
```

Comment:
nit: Let's add it to [`DDConfiguration+apiTests.m`](https://github.com/DataDog/dd-sdk-ios/blob/e2a961ddc9439e523cf234829bd9a27a4d79451d/Tests/DatadogTests/DatadogObjc/ObjcAPITests/DDConfiguration%2BapiTests.m#L92) 🙌.